build/parser.mjs

```diff
@@ -968,9 +978,8 @@ class Parser {
     if (target.EstimatedVaultedDate) {
       item.estimatedVaultDate = this.normalizeOggDate(target.EstimatedVaultedDate);
     }
-    if (target.Vaulted) {
-      item.vaulted = target.Vaulted;
-    }
+
+    item.vaulted = target.vaulted || wikiaItem.vaulted;
```

Comment:
i've started reviewing the data and it seems like this line should merge objects if both are available, we're missing a bunch of 'wikiaThumbnail' entries and other data, also the parent data is all mangled

[itemsOld.json](https://github.com/WFCD/warframe-items/files/13850184/itemsOld.json)
[itemsNew.json](https://github.com/WFCD/warframe-items/files/13850183/itemsNew.json)
`curl https://termbin.com/7yl0`
`jsondiffpatch /itemsOld.json ./itemsNew.json`
if you guys know of a better solution to diff json let me know

Reply:
hmm I'm not entirely sure. Like I said not super versed in js but I don't really see this one would affect wikiaThumbnail (sentence feels a little off but I'm genuinely asking)

Reply:
Sorry my mistake i forgot some entries just are kinda random when you build

Reply:
and also idk why i thought this line was an object it's a boolean, but still some weird thing is happening and valkyr prime doesn't have the vaulted entry anymore

Reply:
That one I will check

Reply:
it's a casing issue i think
```suggestion
    item.vaulted = target.Vaulted || wikiaItem.vaulted;
```

Reply:
i rebuild and it didn't fix it uhhh

Reply:
oh it actually fixed it my mistaked

Reply:
oh well it should be `target.Vaulted` so yeah?

Reply:
Yeah all the data from ogg seems to be in pascalcase